src/BackgroundCells.js

```diff
@@ -62,7 +62,12 @@ class BackgroundCells extends React.Component {
         {range.map((date, index) => {
           let selected =  selecting && index >= startIdx && index <= endIdx;
           return (
-            <Wrapper key={index} value={date}>
+            <Wrapper
+              key={index}
+              value={date}
+              isFirstInRange={index === 0}
+              isLastInRange={index === range.length - 1}
```

Comment:
it makes sense, tho its a bit specific to your needs ;) maybe just pass the range and value in and you can figure out if it's the first or not in your app code? I don't like having props that have no purpose in the code itself, it'll be really easy to break your usecase later without realizing